apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx

```diff
@@ -190,8 +192,7 @@ const SettingsModal: React.FC<React.PropsWithChildren<InjectedModalProps>> = ({
                   />
                   <BetaTag>{t('BETA')}</BetaTag>
                 </Flex>
-
-                <Toggle id="toggle-username-visibility" checked={enabled} scale="md" onChange={toggle} />
+                <WebNotiToggle enabled={enabled} />
```

Comment:
Does it help to lazy load this component? It will always render right?

Reply:
My concern is even if we turn off the web3 notification, we will still load web3inbox related js

Reply:
Still load when this setting is open, not perfect but still good enough

Reply:
ok lgtm. Let's push it out. Please mark as ready to review